modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/tree/BplusTree.java

```diff
@@ -100,9 +100,9 @@
  * <p>All pages in the tree are divided into levels. Leaves are always at the level {@code 0}. Levels of inner pages are thus positive.
  * Each
  * level represents a singly linked list - each page has a link to the <i>forward</i> page at the same level. It can be retrieved by calling
- * {@link BplusIo#getForward(long)}. This link must be a zero if there's no forward page. Forward links on level {@code 0} allow iterating
- * tree's keys and values effectively without traversing any inner nodes ({@code AbstractForwardCursor}). Forward links in inner nodes have
- * different purpose, more on that later.
+ * {@link BplusIo#getForwardPageId(long, int)}. This link must be a zero if there's no forward page. Forward links on level {@code 0} allow
```

Comment:
Old name was good enough, why did you rename it? Please change it back.

Reply:
Fix it